src/lib/core/fiber.h

```diff
@@ -852,6 +852,10 @@ struct cord {
 	struct fiber *main_fiber;
 	/** An event triggered to cancel cord main fiber. */
 	ev_async cancel_event;
+	/** Number of alive client (non system) fibers. */
+	unsigned int client_fiber_count;
```

Comment:
Nit: Please don't use `unsigned` for counters. Use `int`.

Reply:
Fixed.

<details>
<summary>diff</summary>

```diff
--- a/src/lib/core/fiber.h
+++ b/src/lib/core/fiber.h
@@ -853,7 +853,7 @@ struct cord {
        /** An event triggered to cancel cord main fiber. */
        ev_async cancel_event;
        /** Number of alive client (non system) fibers. */
-       unsigned int client_fiber_count;
+       int client_fiber_count;
        /** Fiber calling fiber_shutdown. NULL if there is no such. */
        struct fiber *shutdown_fiber;
 };
diff --git a/test/unit/fiber.cc b/test/unit/fiber.cc
index 4a774420e..82785006e 100644
--- a/test/unit/fiber.cc
+++ b/test/unit/fiber.cc
@@ -586,7 +586,7 @@ fiber_test_client_fiber_count(void)
 {
        header();

-       unsigned int count = cord()->client_fiber_count;
+       int count = cord()->client_fiber_count;

        struct fiber *fiber1 = fiber_new("fiber1", wait_cancel_f);
        fail_unless(fiber1 != NULL);
@@ -634,7 +634,7 @@ fiber_test_set_system(void)

        struct fiber *fiber1 = fiber_new("fiber1", wait_cancel_f);
        fail_unless(fiber1 != NULL);
-       unsigned int count = cord()->client_fiber_count;
+       int count = cord()->client_fiber_count;

        fiber_set_system(fiber1, true);
        fail_unless(--count == cord()->client_fiber_count);
```

</details>